src/main/kotlin/domain/Direction.kt

```diff
@@ -0,0 +1,18 @@
+package domain
+
+enum class Direction(val rowDirection: Int, val columnDirection: Int) {
```

Comment:
```suggestion
enum class Direction(privagte val rowDirection: Int, private val columnDirection: Int) {
```

외부에 노출하지 않는 프로퍼티에 대해서는 접근제어자에 대해서 private이 좋지 않을까요? 🤔